### DISCUSSION TITLE
Eloquent updated event exclude certain columns from triggering event to bust cache

### DISCUSSION THREAD

**Discussion by sts-ryan-holton** [laravel/framework#49636] [Ideas]
Hi, I'm using the eloquent events to flush out cache tags within my Laravel project to improve the user experience whenever a user modifies a model. However, some columns on my model that I've added such as date & time fields aren't updated by the user, yet the `updated` event will still run thus invalidating my cache all of the time.

I'd like to be able to exclude certain columns from the updated event so that they don't trigger the function, here's my current set up:

```php
/**
 * Bootstrap any application services.
 */
public static function boot()
{
    parent::boot();

    // on create
    static::created(function ($model) {
        Cache::tags('affiliates')->flush();
    });

    // on update
    static::updated(function ($model) {
        // TODO: somehow exclude some columns so that flush() doesn't run
        Cache::tags('affiliates')->flush();
    });

    // on delete
    static::deleted(function ($model) {
        Cache::tags('affiliates')->flush();

        $model->products()->delete();
        $model->splits()->delete();
        $model->dataTransformers()->delete();
        $model->campaigns()->delete();
    });
}
```

**Comment by macropay-solutions** [comment]
Use
   if (!$model->wasChanged(['column'])) {
      return;
  }
